extmod/modnetwork.c

```diff
@@ -91,8 +91,10 @@ STATIC const mp_rom_map_elem_t mp_module_network_globals_table[] = {
     MICROPY_PORT_NETWORK_INTERFACES
 
     // Constants
+    #if MICROPY_PY_NETWORK_CYW43
```

Comment:
These constants are also needed if the ninaw10 driver is enabled.  It might be better to just leave them here unconditionally.